### PR TITLE
setting options in ls and sending div ids

### PR DESCRIPTION
**Pull request by nishlashakya**
@Anthony-Michel

### CODE REVIEW

File: utils/utils.js [hunk 1]
```diff
@@ -105,17 +103,18 @@ async function createDocumentHandler() {
 			docCompletionButton: document.getElementById('docCompletionButton').value || 'Complete DBQ'
 		}
 		
-		const doc = await zegal.createDocument(doctypePayload, options)
+		const doc = await zegal.createDocument(doctypePayload, options, 'anyModalBodyId')
```
Comment (Anthony-Michel) [inline utils/utils.js]: what's that for ?

Comment (nishlashakya) [reply utils/utils.js]: @Anthony-Michel we are sending div's id to zegal-client

File: utils/utils.js [hunk 2]
```diff
@@ -105,17 +103,18 @@ async function createDocumentHandler() {
 			docCompletionButton: document.getElementById('docCompletionButton').value || 'Complete DBQ'
 		}
 		
-		const doc = await zegal.createDocument(doctypePayload, options)
+		const doc = await zegal.createDocument(doctypePayload, options, 'modalBodyId')
```
Comment (Anthony-Michel) [inline utils/utils.js]: this value is still hardcoded here.
Are you not suppose to get it back as parameter from somewhere ?

Comment (nishlashakya) [reply utils/utils.js]: Taking API portal as 3rd party client, this is dev who is sending their modal div-id, where we will show our application. Previously zegal-client had that hard coded; now 3rd party dev sends their div id @Anthony-Michel

Comment (nishlashakya) [reply utils/utils.js]: This was my understanding when HC told, 3rd party dev needs to set id not us. Please correct me if I am wrong
@Anthony-Michel

**Review by Anthony-Michel — APPROVED** [review]

**Pull request merged**